submissions/AnthonyShostak/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head lang="EN">
+        <title>Popup</title>
+        <link rel="stylesheet" href="css/style.css"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+    </head>
+    <body>
+        <header class="header">
+            <nav class="nav__header">
+                <ul class="nav__list">
+                    <li class="nav__item mail"><a href="#">Mail</a></li>
+                    <li class="nav__item images"><a href="#">Images</a></li>
+                    <li class="nav__item popup__nav__item">
+                        <input type="checkbox" id="popup" class="popup_checkbox">
+                        <label for="popup">
+                            <img src="popup-icons/popup-button.png" alt="popup-button" class="icon popup__icon">
+                        </label>
+                        <div class="popup">
+                            <ul class="popup__items">
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/cart.png" alt="cart" class="icon">Cart</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/chat.png" alt="chat" class="icon">Chat</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/favorites.png" alt="favourites" class="icon">Favourites</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/home.png" alt="home" class="icon">Home</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/games.png" alt="games" class="icon">Games</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/location.png" alt="location" class="icon">Location</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/mic.png" alt="microphone" class="icon">Microphone</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/night-mode.png" alt="night-mode" class="icon">Night mode</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/scan.png" alt="scan" class="icon">Scan</a></li>
+                                <li class="more__below">
+                                    <input type="checkbox" id="popup__more" class="checkbox__more">
+                                    <label for="popup__more" class="more">Press to see more</label>
+                                    <label for="popup__more" class="less">Press to see less</label>
```

Comment:
Add `cursor: pointer` to indicate to users about an interactive element.
Logically when the second part of the popup is open this button should be hidden.

Reply:
Added cursor: pointer to interactive elements.
Removed "Press to see less" button.
Also "Press to see more" button is now hidden after it's clicked.